cmd/process-agent/main_common.go

```diff
@@ -163,6 +164,11 @@ func runAgent(exit chan struct{}) {
 	agentVersion, _ := version.Agent()
 	log.Infof("running version: %s", agentVersion.GetNumberAndPre())
 
+	fmt.Println("helloo")
```

Comment:
nit: make sure you don't check print statements into source 😄

Reply:
oops 😅